4/README.md

```diff
@@ -0,0 +1,207 @@
+- docker-compose.benchmark.yml
+  - DEPRECATED: Used to deploy standalone BigchainDB v1.3 development environment.
+
+
+Majority of the above mentioned files are either not in use OR can be integrated in a way that all of them are being tested effectively with CI pipeline(s).
+
+We also have multiple Dockerfiles in the repository:
+
+- Dockerfile
+  - Dockerfile used to build and publish BigchainDB production images.
+- Dockerfile-dev
+  - Dockerfile used for development purposes.
+- compose/bigchaindb-driver/Dockerfile
+  - Dockerfile used to spawn a container in development environment with bigchaindb-driver installed.
+    - This seems to be an invalid usecase for a separate container. In my opinion, `bigchaindb-driver` does not qualify as a service. It is just a python-package that can reside on a any of the containers, preferably BigchainDB.
+- compose/bigchaindb-server/Dockerfile
+  - This is a duplicate of `Dockerfile-dev`
+- compose/travis/Dockerfile
+  - Dockerfile used to build/run BigchainDB service in CI.
+
+## Motivation
+There are multiple problems with our current workflow and directory structure which also impacts testing:
+
+- We need to deprecate or remove the unused or redundant docker-compose and Docker file(s).
+- Since our development and CI files are different, there is no way to test the development `docker-compose` file(s). There are two solutions to this:
+  - Write tests to test the docker-compose for development.
+  - Integrate the development and CI workflow.
+
+
+## Specification
+
+### Proposed Solution
+
+- Clean up the deprecated workflow(s).
+  - Remove `docker-compose.rdb.yml`.
+  - Remove `docker-compose.benchmark.yml`.
+  - Remove `docker-compose.docs.yml`, since building documentation should be part of a development environment.
+  - Remove `docker-compose.ssl.yml`, because we are planning to remove SSL support for MongoDB.
+  - Remove `docker-compose.network.yml` file, this workflow will be supported by another deployment method. Currently, we are not using this method for development. We will have Vagrant, Ansible and K8s network driver to offer this functionality.
+
+- Update the workflows with the new deployment workflow(s).
+  - Update `docker-compose.yml` to support the current deployment strategy i.e. BigchainDB + Tendermint + Stand alone MongoDB.
+- Integrate the development and CI compose file(s) in such a way that there are minimal differences and they can be tested on a CI, also rename the files:
+  - `docker-compose.yml` should have the base services i.e.
+  BigchainDB, MongoDB, Tendermint and any tools on top can be made part of `docker-compose.dev.yml`
+    - docker-compose.travis.yml -> docker-compose.yml
+    - docker-compose.tendermint.yml -> docker-compose.dev.yml
+
+- Update the MongoDB and Tendermint version(s) to the latest supported version:
+
+  - mongo:3.4.13
+  - tendermint/tendermint:0.13
```

Comment:
since we are still in development phase might as well go with latest tendermint version

Reply:
As I write this, I think the latest Tendermint release is actually 0.16.0 so using `latest` could get confusing, since we don't support _that_. We support the latest that `py-abci` supports, as I understand it.

Reply:
We had problems with the `latest` tendermint. The changes were not reflected in the `py-abci` and I don't know the current status of this.

Reply:
In that case, I will stick to `0.13.0` because we are currently testing with that.